src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForPathAndValueEntity.cs

```diff
@@ -175,12 +178,16 @@ protected IMoBiCommand EditFormulaAndSetOnBuilder(TBuildingBlock buildingBlock,
             modalPresenter.Encapsulate(editFormulaPresenter);
             modalPresenter.Text = AppConstants.Captions.EditFormula;
             var usingFormulaDecoder = new UsingFormulaDecoder();
-            editFormulaPresenter.Init(builder, buildingBlock, usingFormulaDecoder);
+
+            var clonedBuildingBlock = _cloneManagerForBuildingBlock.Clone(buildingBlock);
+            var clonedBuilder = clonedBuildingBlock.ByPath(builder.Path);
+            editFormulaPresenter.InitializeWith(new MoBiMacroCommand());
+            editFormulaPresenter.Init(clonedBuilder, clonedBuildingBlock, usingFormulaDecoder);
```

Comment:
and here we aer editing the clone. I think a comment would help understand. I can't remember why it's like this and for sure why we are now cloning